bin/ember-template-lint.js

```diff
@@ -57,6 +57,20 @@ function printErrors(errors, invocationOptions) {
       );
     }
   }
+
+  if (process.env.GITHUB_ACTIONS) {
+    let { issueCommand } = require('@actions/core/lib/command');
```

Comment:
I'm a bit worried about requiring deeply like this. 🤔 

https://github.com/actions/toolkit/blob/1c12ced7ba4b5f2d21da0ddfea83b4885505503f/packages/core/src/command.ts#L3

Does not help assuage my concern...

Reply:
indeed, but unfortunately there is no other way to include file metadata in the message 😞 

if we don't use this lib then we'll have to construct the `::error file={name},line={line},col={col}::{message}` message ourselves 🤔

Reply:
It seems like they support emitting the console output directly as public API [see here](https://github.com/actions/toolkit/blob/master/docs/commands.md), though they don't document the specific output that `issueCommand` emits here.

I guess I'm happy to do it this way for now, but I would like to see if they can either export `issueCommand` as public API or document the `::error` syntax ([on this page](https://github.com/actions/toolkit/blob/master/docs/commands.md)) so we can just use `console.log`.

Reply:
🤦‍♂ I totally missed your link in the PR description which shows public documentation of the format.

@Turbo87 - Can you update this to drop the additional dependency, and directly `console.log` the appropriate format when `process.env.GITHUB_ACTIONS` is set?

Reply:
yep, can do. but I think I remember that their tool did some escaping, which we would probably have to do too then.

Reply:
Ya, I think since we know the line, col, and filename will not have any of the special chars they escape ([see here](https://github.com/actions/toolkit/blob/1c12ced7ba4b5f2d21da0ddfea83b4885505503f/packages/core/src/command.ts#L81-L87)) we don't have to worry about those ones, but we will need to escape any newlines in the message part.

I think the result would be something like:

```js
let message = error.message.replace(/\r/g, '%0D').replace(/\n/g, '%0A');

console.log(`::error file={${file}},line={${line}},col={${col}}::${message}`);
```

Reply:
done